model-optimizer/extensions/ops/dequantize_linear.py

```diff
@@ -0,0 +1,42 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import numpy as np
+
+from mo.graph.graph import Node, Graph
+from mo.ops.op import Op
+
+
+class DequantizeLinear(Op):
+    op = 'DequantizeLinear'
+    enabled = True
+
+    def __init__(self, graph: Graph, attrs: dict):
+        mandatory_props = {
+            'type': None,
+            'op': __class__.op,
+            'version': None,
+            'infer': None,
+            'on_value': None,
+            'off_value': None,
+            'out_ports_count': 1,
+            'in_ports_count': 3,
+            'data_type': None,
+            'type_infer': None,
```

Comment:
The operation will not survive until the type inference, so we can remove this line.